src/components/DismissablePane.tsx

```diff
@@ -1,14 +1,19 @@
 import React, { Dispatch, SetStateAction } from 'react';
 import { Headline, Pane, PaneHeader } from '@folio/stripes/components';
+import { Button } from '@material-ui/core';
 
 interface DismissablePaneProps {
     displayed: boolean;
-    setDisplayed: Dispatch<SetStateAction<boolean>>;
+    setDisplayed?: Dispatch<SetStateAction<boolean>>;
 }
 
 const DismissablePane = (props: DismissablePaneProps) => {
   function dismissPane() {
-    props.setDisplayed(false);
+    try {
```

Comment:
Why did this get added?  You shouldn't need anything more than the dismiss function (although I'd recommend making it a react callback `const dismissPane = useCallback(() => props.setDisplayed(false), [props])`)

Reply:
Ok, that was from when I was trying to find a way to test the state and I made the setDisplayed property of the interface optional. And so to fix that I added those try catch to handle cases when it is undefined. 

But now that I can use `const setDisplayed = jest.fn();` for my useState, I fixed it and its no longer optional and also made the function a react callbacl.

Reply:
Gotcha! ye, `jest.fn` is great; there's more on it in the next module, too

Reply:
Makes it really easy to test callbacks and simulate functions.  You can also do more complex things like having it return different values throughout the test, etc